### DISCUSSION TITLE
Issue with CoPilot student verification

### DISCUSSION THREAD

**Discussion by amulifts** [community/community#43637] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Dear CoPilot Support Team,

I am writing to request assistance with an issue I am experiencing with CoPilot. I am a verified student, but when I try to use the service, I am being asked to purchase a membership. I should not have to pay for a membership as a verified student, so I believe there may be an issue with my account.

Could you please help me resolve this issue and gain access to CoPilot as a verified student? I have attached proof of my student status for your reference.

Thank you for your help and I look forward to your response.

Sincerely,
Aman Khadka

**Comment by jongwooo** [comment]
@amulifts [Others](https://github.com/community/community/discussions/19050) are also raising similar issues. I hope it is resolved quickly.

**Comment by otkrickey** [comment]
Solved: #43634
